cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -127,20 +132,37 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		}
 	} else if !isWCOW {
 		return nil, errors.Wrap(errdefs.ErrFailedPrecondition, "oci spec does not contain WCOW or LCOW spec")
+	} else if oci.IsJobContainer(s) {
+		// If privileged was specified just fake a task (i.e reuse the wcowPodSandbox logic)
```

Comment:
does this mean there isn't a sandbox pod? If that is the case we won't be able to use `kubectl port-forward` since the sandbox pod uses wincat.exe for the portforwarding. ref: https://github.com/kubernetes/kubernetes/pull/75479

Reply:
Correct (and this also mentions the word privileged still :P haha). Do we need port forwarding if everything's on the host anyways and in the default network compartment?